core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -275,7 +275,7 @@ class CacheHeaders extends Audit {
       {key: 'url', valueType: 'url', label: str_(i18n.UIStrings.columnURL)},
       // TODO(i18n): pre-compute localized duration
       {key: 'cacheLifetimeMs', valueType: 'ms', label: str_(i18n.UIStrings.columnCacheTTL),
-        displayUnit: 'duration'},
+        displayUnit: 'duration', dontAggregate: true},
```

Comment:
What if we make this `aggregate: false` or `aggregateEntities: false` and then give it a default value of `true`?

Reply:
That would result in additional LHR payload for every column, right? The default falsy and when omitted works to reduce transport.

Reply:
I'm saying that `aggregate: undefined` should be treated the same as `aggregate: true` in this scenario, though making falsy values work this same can also be valuable.

I guess the name `dontAggregate` bugs me because of the don't/dont business. I could also see `noAggregation` or `preventAggregation` working.